torch/__init__.py

```diff
@@ -42,7 +42,6 @@
 ################################################################################
 
 if platform.system() == 'Windows':
```

Comment:
Can you try this to make `mypy` understand the use of `ctypes.WinDLL` (according to https://mypy.readthedocs.io/en/stable/common_issues.html#python-version-and-system-platform-checks )
```suggestion
if sys.platform in ('win32', 'cygwin'):
```

Reply:
Interesting fact that it only understands `sys.platform`.